src/particle/corsika/gen_conversion_table.py

```diff
@@ -0,0 +1,182 @@
+# Copyright (c) 2018-2022, Eduardo Rodrigues and Henry Schreiner.
+#
+# Distributed under the 3-clause BSD license, see accompanying file LICENSE
+# or https://github.com/scikit-hep/particle for details.
+"""
+Function to generate .csv conversion table from corsica to pdg_id and vice-versa.
```

Comment:
Nicer if you are explicit about the name of the CSV file being produced and use the name PDGID, etc.